serverless.yml

```diff
@@ -82,6 +83,12 @@ functions:
       - httpApi:
           path: /{email}/close-first-steps
           method: post
+  deleteFlags:
+    handler: src/deleteFlags.deleteFlags
+    events:
+      - httpApi:
+          path: /{email}/delete-flags
```

Comment:
In my opinion, this is the right path:

```suggestion
          path: /{email}
```

What do you think?

Reply:
Makes sense